Правда или действие (часть 2)

Alexey-og opened the pull request:
… комментариев

Выявлена, но не решена следующая проблема:
При просмотре в попапе информации первой фотографии, у которой количество комментариев больше пяти, показывается всё правильно. При просмотре второй фотографии с количеством комментариев больше пяти, первые пять комментариев показываются правильно, но последующие комментарии берутся от просмотренной до этого фотографии. При этом в функцию "showPopup" в параметры попадает правильный массив комментариев .

---
:mortar_board: [Правда или действие (часть 2)](https://up.htmlacademy.ru/javascript/25/user/10069/tasks/15)

:boom: https://htmlacademy-javascript.github.io/10069-kekstagram-25/8/

Commit:
Добавлена работоспособность кнопки "Загрузить еще" по догрузке списка комментариев

Выявлена, но не решена следующая проблема:
При просмотре в попапе информации первой фотографии, у которой количество комментариев больше пяти, показывается всё правильно. При просмотре второй фотографии с количеством комментариев больше пяти, первые пять комментариев показываются правильно, но последующие комментарии берутся от просмотренной до этого фотографии. При этом в функцию "showPopup" в параметры попадает правильный массив комментариев .

senchkim commented:
> … комментариев
> 
> Выявлена, но не решена следующая проблема: При просмотре в попапе информации первой фотографии, у которой количество комментариев больше пяти, показывается всё правильно. При просмотре второй фотографии с количеством комментариев больше пяти, первые пять комментариев показываются правильно, но последующие комментарии берутся от просмотренной до этого фотографии. При этом в функцию "showPopup" в параметры попадает правильный массив комментариев .
> 
> 🎓 [Правда или действие (часть 2)](https://up.htmlacademy.ru/javascript/25/user/10069/tasks/15)
> 
> 💥 https://htmlacademy-javascript.github.io/10069-kekstagram-25/8/

У тебя должно быть так при старте приложения:
При рендере карточек у каждой карточки должен быть свой набор комментариев - то есть они должны быть независимыми друг от друга!

Commit:
Исправление ошибочной работы кнопки "Загрузить еще".

Ошибка была в не удалении обработчика события по нажатию по кнопке.

Модуль "modal-big-picture.js"  требует рефакторинга.
